pkg/agent/datamodel/types.go

```diff
@@ -1818,11 +1818,26 @@ type KubeletWebhookAuthorization struct {
 	CacheUnauthorizedTTL Duration `json:"cacheUnauthorizedTTL,omitempty"`
 }
 
-type VMSSInstanceViewCSEStatus struct {
-	// ExitCode stores the exitCode from VMSS CSE output.
+type CSEStatus struct {
+	// ExitCode stores the exitCode from CSE output.
 	ExitCode string `json:"exitCode,omitempty"`
-	// Output stores the output from VMSS CSE output.
+	// Output stores the output from CSE output.
 	Output string `json:"output,omitempty"`
-	// Error stores the error from VMSS CSE output.
+	// Error stores the error from CSE output.
 	Error string `json:"error,omitempty"`
+	// ExecDuration stores the execDuration from CSE output.
+	ExecDuration int `json:"execDuration,omitempty"`
+}
+
+type Error struct {
+	Code    string
```

Comment:
Let's give it a type alias 
type CSEStatusParsingErrorCode string
Code CSEStatusParsingErrorCode 

And define the const as this dedicated type so consumers can do comparision
const CSEMessageUnmarshalError CSEStatusParsingErrorCode = "CSEMessageUnmarshalError "